shaman/util.py

```diff
@@ -55,3 +55,54 @@ def is_node_healthy(node):
     node.down_count = 0
     models.commit()
     return True
+
+
+def parse_distro_release(identifier):
+    """
+    Back and forth translation for a release identifier, falling back to
+    ``None`` when an identifier has no existing mapping.
+
+    For CentOS, an identifier of '7' will return: None, '7'.
```

Comment:
I think the codename for CentOS is ``core``. Should we return that even though every CentOS release has the same codename?